etl/steps/data/garden/unesco/2024-06-25/education_sdgs.py

```diff
@@ -0,0 +1,111 @@
+"""Load a meadow dataset and create a garden dataset."""
+from owid.catalog import VariableMeta
+from tqdm import tqdm
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("education_sdgs")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["education_sdgs"].reset_index()
+
+    # Retrieve snapshot with the metadata provided via World Bank.
+
+    snap_wb = paths.load_snapshot("edstats_metadata.xls")
+    tb_wb = snap_wb.read()
+
+    #
+    # Process data.
+    #
+    tb = geo.harmonize_countries(
+        df=tb, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
+    )
+
+    # Drop columns that are not needed
+    tb = tb.drop(columns=["magnitude", "qualifier"])
+
+    # Add the long description from the World Bank metadata
+    long_definition = {}
+    for indicator in tb["indicator_label_en"].unique():
+        definition = tb_wb[tb_wb["Indicator Name"] == indicator]["Long definition"].values
+        if len(definition) > 0:
+            long_definition[indicator] = definition[0]
+        else:
+            long_definition[indicator] = ""
+
+    tb["long_description"] = tb["indicator_label_en"].map(long_definition)
+    # Drop rows with missing indicator labels
+    tb = tb[tb["indicator_label_en"].notna()]
+    tb["indicator_label_en"] = tb["indicator_label_en"].astype(str) + ", " + tb["indicator_id"].astype(str)
+
+    # Pivot the table to have the indicators as columns to add descriptions from producer
+    tb_pivoted = tb.pivot(index=["country", "year"], columns="indicator_label_en", values="value")
+
+    for column in tqdm(tb_pivoted.columns):
+        meta = tb_pivoted[column].metadata
+        meta.display = {}
+        long_definition = tb["long_description"].loc[tb["indicator_label_en"] == column]
+        if not long_definition.empty:
+            long_definition = long_definition.iloc[0]
+            meta.description_from_producer = long_definition
+            meta.title = column
+        #
+        # Update metadata units, short_units and number of decimal places to display depending on what keywords the variable name contains
+        #
+        # Fill out units and decimal places
+        if "%" in column.lower():
```

Comment:
[This variable](http://staging-site-unesco-ed-sdgs/admin/variables/947103) doesn't seem to get captured by these conditions